### PR TITLE
feat(pie): add outRadius setting for pie chart

### PR DESCRIPTION
**Pull request by jianArthur**
## Description

> Add outRadius setting for pie chart

## Screenshots

image

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation or example update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation and example

### CODE REVIEW

**Review by watermelon-pan — APPROVED** [review]
LGTM

File: src/charts/pie/pie.ts [hunk 1]
```diff
@@ -125,9 +125,10 @@ export class PieChart<TData extends PieData, TOptions extends PieChartOptions> e
     };
     this.enableSegmentClickable();
     this.enableLegend();
-    let options: ChartOptions = {
+    let options: ChartOptions & { radius?: number | string } = {
```
Comment (bndynet) [inline src/charts/pie/pie.ts:128]: Can we use `PieChartOptions` instead of `ChartOptions` here?

Comment (bndynet) [reply src/charts/pie/pie.ts:128]: We should fix the lint errors as much as possible, rather than using workaround instead.

Comment (bndynet) [reply src/charts/pie/pie.ts:128]: Got it, here are the chartjs options.

**Comment by bndynet** [comment]
@jianArthur  Please update the docs

**Review by bndynet — APPROVED** [review]

**Pull request merged**